### PR TITLE
Add shortcut to view page source

### PR DESCRIPTION
**Pull request by Sh1d0w**
Added handy shortcut to display page source, just like in Google Chrome.

cc @bbondy @bridiver @bsclifton

### CODE REVIEW

File: js/components/frame.js [hunk 1]
```diff
@@ -452,7 +452,10 @@ class Frame extends ImmutableComponent {
         break
       case 'view-source':
         const sourceLocation = UrlUtil.getViewSourceUrlFromUrl(this.webview.getURL())
-        windowActions.newFrame({location: sourceLocation}, true)
+        let src = this.frame.get('src')
+        if (UrlUtil.isHttpAddress(src) && !UrlUtil.isImageAddress(src)) {
```
Comment (bridiver) [inline js/components/frame.js]: can we split this change out from the shortcut? I think we probably want to make these extra checks in `getViewSourceUrlFromUrl` and maybe return a null location if there is not valid view-source for the url
@bradleyrichter is this shortcut in your list?

Comment (Sh1d0w) [inline js/components/frame.js]: Good point, moved.

**Comment by bbondy** [comment]
If you type in gmail compose, or any text editable for example, and do command+U to underline it, it'll open the page source in a new tab instead.  

I tweaked it to be the same shortcut as Chrome (with an Alt key too and merged. Thanks!)

**Comment by Sh1d0w** [comment]
@bbondy I think you forgot to put the check for the current address like I did in my PR, so if you click CTRL + ALT + U on new tab frame or on a image you currently opened in a browser you will trigger view-source, which does not happend on other browsers and should not happen in Brave as well. 

Please see my PR for reference how I have checked this, so you can put that too :)

**Comment by bbondy** [comment]
The task was merged so it is all commits
Is there a problem on master?

**Comment by bbondy** [comment]
I only merged your PR and then did a commit on top.

**Comment by Sh1d0w** [comment]
@bbondy Ah never mind my comment I tought you closed my PR without merging it because github stated `Closed with unmerged commits`. It's all ok.

**Pull request closed without merging**